src/lib.rs

```diff
@@ -282,6 +282,14 @@ pub struct TxOut {
     address: String,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum KeychainKind {
+    /// External keys.
+    ExternalKeys = 0,
```

Comment:
Why do these enums require numbers?

Reply:
In rust it allows you to de-reference the enum to this number, which I think is used when you want to create a new HD path and use 0 for your external path, and 1 for your internal.  I left it here uncase we want to do something similar in the future. 

Also I saw  your note on the `uniffi-rs` project about this issue and I think it's worth waiting on this PR to see what they say about fixing the issue on their side, or at least if they say it's possible.